src/modules/other-features/OtherFeatureDetail.js

```diff
@@ -238,11 +238,11 @@ const OtherFeatureDetail = (props) => {
               </ListItem>
               <FlatListItemSeparator/>
               {isOtherType() && (
-                <ListItem containerStyle={commonStyles.listItemMinimalVerticalPadding}>
+                <ListItem containerStyle={commonStyles.listItem}>
                   <ListItem.Title style={formStyles.fieldLabel}>{'Other Feature Type'}</ListItem.Title>
```

Comment:
This needs to be a field label style though its list item, because it has to be in bold. Only exception in this case.